### PR TITLE
Feat: tweaking prompts

### PR DESCRIPTION
**Pull request by aeschi**

### CODE REVIEW

File: services/rest/paths/print/idea.js [hunk 1]
```diff
@@ -34,9 +46,13 @@ export async function getIdea() {
 		return "error";
 	}
 
-	const idea = choices[0].message.content;
+	const idea = JSON.parse(choices[0].message.content);
```
Comment (raphael-arce) [inline services/rest/paths/print/idea.js]: suggestion, let's destructure:
```js
const { product_idea, image_prompt } = JSON.parse(choices[0].message.content);
```

File: services/rest/paths/print/idea.js [hunk 2]
```diff
@@ -34,9 +46,13 @@ export async function getIdea() {
...
 
-	// const idea = `"ParkFit": Ein smartes Armband, das die Bewegung der Berliner:innen
-	// 	im Volkspark Friedrichshain überwacht und sie mit gesunden SnackEmpfehlungen belohnt, basierend auf ihrem Aktivitätslevel.`;
+	// const idea = {
+	// 	product_idea:
+	// 		"Hey Vergangenheits-Ich! Ich habe die perfekte Idee für ein lebenswerteres Berlin: Ein fliegender Teppich-Verleihservice! Schwebend über den Straßen Berlins kannst du Staus umgehen und die Stadt aus der Vogelperspektive entdecken. Einfach magisch, oder? 🧞‍♂️✨",
+	// 	image_prompt:
+	// 		"A futuristic image of Berlin with flying carpets gliding above the streets, transporting people smoothly through the city. The magical scene includes colorful carpets with intricate designs, adding a touch of whimsy to urban mobility.",
+	// };
 	return { idea, focusGroup, topic, medium };
```
Comment (raphael-arce) [inline services/rest/paths/print/idea.js]: after destructuring above, imo we don't need to further nest, we can "directly" return it:
```js
return { product_idea, image_prompt, focusGroup, topic, medium } 
```
this way we also prevent nesting (`idea.idea.product_idea`) in the `createPostcardHtml()` function.

nitpick: would be good if the name `product_idea`and `image_prompt` follow the camelCase pattern.

**Review by raphael-arce — APPROVED** [review]
one suggestion, otherwise lgtm 👍

File: services/rest/paths/print/idea.js (hunk 2 above)
Comment (aeschi) [reply services/rest/paths/print/idea.js]: yes, good idea!

**Pull request merged**